### PR TITLE
Upgrade group resource - part 4

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves provider upgrade

Test passes on fork repo: https://github.com/vmanilo/terraform-provider-twingate/actions/runs/5825472662

## Changes
- upgrade group resource

### CODE REVIEW

File: docs/data-sources/groups.md [hunk 1]
```diff
@@ -26,16 +26,16 @@ data "twingate_groups" "foo" {
...
+- `groups` (Attributes List) List of Groups (see [below for nested schema](#nestedatt--groups))
 - `is_active` (Boolean) Returns only Groups matching the specified state.
 - `name` (String) Returns only Groups that exactly match this name.
 - `type` (String) Returns only Groups of the specified type (valid: `MANUAL`, `SYNCED`, `SYSTEM`).
 
 ### Read-Only
 
-- `groups` (Block List) List of Groups (see [below for nested schema](#nestedblock--groups))
-- `id` (String) The ID of this resource.
+- `id` (String) The ID of the data source result
```
Comment (vmanilo) [inline docs/data-sources/groups.md:36]: @ekampf should I revert this changes?

**Review by ekampf — APPROVED** [review]
